libs/sdk-ui-ext/src/internal/utils/embeddingInsightViewCodeGenerator/insightViewCodeGenerator.ts

```diff
@@ -18,9 +17,10 @@ export const insightViewCodeGenerator = getReactEmbeddingCodeGenerator<IInsightV
         package: "@gooddata/sdk-ui-ext",
     },
     insightToProps: (insightDefinition, ctx) => {
+        const id = insightId(insightDefinition as IInsight);
         return {
             insight: {
-                value: insightRef(insightDefinition as IInsight),
+                value: id,
                 meta: {
                     typeImport: {
```

Comment:
This is no longer ObjRef, but string, so the `typeImport` prop can safely be removed.

Reply:
done